CHANGELOG.md

```diff
@@ -1,3 +1,254 @@
+# 23.10 (2023-10-13)
+
+A big shout out to all the contributors! We had 118 contributors in this release.
+
+Breaking changes:
+
+- Support multiple language servers per language (#2507)
+    - This is a breaking change to language configuration
+- Maintain the cursor position and view when splitting with `:hsplit`/`:vsplit` (#8109)
+    - Previously, splitting would open the new buffer at the top of the file
```

Comment:
hmm is this really a breaking change?

I would only call things that break configuration a breaking change

Reply:
Hmm yeah I suppose behavior changes aren't necessarily breaking changes. I like the idea of using breaking changes just for configuration or renames 👍